### PR TITLE
Feature answer list answers

### PR DESCRIPTION
**Pull request by brynrmrzz**
A few tickets in this PR. Everything is coming together!

https://trello.com/c/gRa0ezfT/33-questionanswerlist

Some styling from this ticket as well
https://trello.com/c/ojpDSQTG/83-shared-scss-set-global-bg-primary-bg-secondary-font-primary-font-secondary

### CODE REVIEW

File: src/components/questions/Answer.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <span> | </span>
+        <a> Report </a>
+      </div>
+      <div className="photo-container">
+        {
+          (answer.photos)?
+          (answer.photos.map((photo) => {
+              return <img key={ photo.id } src={ photo.url } />;
+          }))
+          :null
```
Comment (lbc1013) [inline src/components/questions/Answer.jsx:29]: In my opinion, I think for the conditional rendering, when the condition is false and there's nothing you want to render for the false condition like null, you can use as the below. 

You don't have to change but might consider next time. I personally use as the below when I want to render only if the condition is true.

{(condition) && (result, only if the condition is true)}

Example :
{(answer.photos) && (
      (answer.photos.map((photo) => {
          return ;
      }))
}

File: src/components/questions/AnswerList.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <br></br>
+      {
+        (answers.length > 2)?
+          (<a
+            id="see-more-answers-btn"
+            onClick={()=>{
+              loadMoreAnswers()
+            }}
+            > LOAD MORE ANSWERS </a>)
+          :null
```
Comment (lbc1013) [inline src/components/questions/AnswerList.jsx]: same as the above!

**Review by lbc1013 — APPROVED** [review]
Looks good to me other than the comment! but I think it's just my preference. 🤔

File: src/components/questions/Answer.jsx (hunk 1 above)
Comment (brynrmrzz) [reply src/components/questions/Answer.jsx:29]: Like this a lot!

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,7 @@
 node_modules
 dist/bundle.js
 src/config.js
-package-lock.json
\ No newline at end of file
+package-lock.json
+src/components/overview
```
Comment (kjniemela) [inline .gitignore]: Why are these folders on the gitignore?

File: src/components/questions/SearchBar.jsx [hunk 4]
```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 const SearchBar = (props) => {
   return (
-    <div id="search-bar-component">
-      <label name='question-search'></label>
+    <div id="search-bar-container">
+      <label name='question-search'>QUESTION & ANSWERS</label>
```
Comment (kjniemela) [inline src/components/questions/SearchBar.jsx:6]: I think '&' is a html special character, it should probably be replaced by '&amp;'

**Review by kjniemela — CHANGES_REQUESTED** [review]

**Pull request merged**